strict.js

```diff
@@ -0,0 +1,78 @@
+module.exports = {
+  root: true,
```

Comment:
And actually, I'm thinking we could follow the same approach for `web` and `react` presets, making them just be files in this repo; so we avoid having multiple repos and then having a `web` and `react` hierarchy is less of a concern.

Will have to explore how to extend / apply the hierarchy within the same repo between JS files

Reply:
> making them just be files in this repo

I think this would generate a problem with dependencies and peer dependencies. Probably the `react` preset will have peer dependencies that are not needed by the node preset.

That is why we decided to split things into separate repos, so we could better control peer dependencies. 

With our internal modules we ended up having to install `react` in a node.js project that didn't needed and that was very confusing.

Reply:
Update after all the final changes:

- The 2 presets that remain in the repo (defaul vs legacy) have `root: true` line, as they don't extend each other.
- On the other hand, we have moved all the React-related config to the `eslint-config-react` repository: https://github.com/open-turo/eslint-config-react/pull/85